tasks/eth/3-add-superchainConfig/justfile

```diff
@@ -4,47 +4,115 @@ export rpcUrl := env_var('ETH_RPC_URL')
 export signatures := env_var_or_default('SIGNATURES', '')
 export bundle := "input"
 export location := `pwd`
-export OWNER_SAFE := env_var('FOUNDATION_SAFE')
 
-simulate hdPath='0':
+# Accounts
+export councilSafe := env_var('COUNCIL_SAFE')
+export foundationSafe := env_var('FOUNDATION_SAFE')
+export ownerSafe := env_var('FOUNDATION_SAFE')
+export randomPersonEoa := "0xd8dA6BF26964aF9D7eEd9e03E53415D37aA96045"
+
+
+simulate-council hdPath='0':
+  #!/usr/bin/env bash
+  cd ../../..
+  echo "getting signer address..."
+  if [ t"$SIMULATE_WITHOUT_LEDGER" == t1 ]
+  then
+    signer="0x07dC0893cAfbF810e3E72505041f2865726Fd073"
+    echo "Simulating with first owner: ${signer}"
+  else
+    signer=$(cast wallet address --ledger --mnemonic-derivation-path "m/44'/60'/{{hdPath}}'/0/0")
+    echo "Simulating with leger account: ${signer}"
+  fi
+  echo ""
+  forge script NestedSignFromJson \
+    --rpc-url ${rpcUrl} \
+    --sender ${signer} \
+    --sig "signJson(string,address)" \
+    "${location}/${bundle}.json" \
+    "${councilSafe}"
+
+simulate-foundation hdPath='0':
```

Comment:
Do we intentionally duplicate the logic instead of adding a new `safeKind = foundation | council` parameter to avoid any breaking changes to UX? https://just.systems/man/en/chapter_38.html

Reply:
I had a bit of difficulty getting the `just` syntax to work for that, so I just did this, but I agree it's worth investigating further.

Reply:
Tracking in https://github.com/ethereum-optimism/superchain-ops/issues/80, will resolve this